utils.ts

```diff
@@ -350,9 +350,12 @@ export async function buildVitest({ verify = false }) {
 	cd(vitestPath)
 	const frozenInstall = getCommand('pnpm', 'frozen')
 	const runBuild = getCommand('pnpm', 'run', ['build'])
+	const runPack = `pnpm -r --filter=./packages/** exec -- pnpm pack`
```

Comment:
The `getCommand` from `@antfu/ni` didn't support this so hardcoding the command here.